src/test/ui/duplicate/dupe-symbols-7.stderr

```diff
@@ -6,5 +6,7 @@ LL | fn main(){}
    |
    = help: did you use #[no_mangle] on `fn main`? Use #[start] instead
 
+thread '<unnamed>' panicked at 'Metadata module not compiled?', src/libcore/option.rs:1034:5
+note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

Comment:
This here looks like an actual issue, we shouldn't get a panic here, should we?

This [reproduces on playground](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=e5f0c62b43e0f4cab359b4bcf34f1e03), but it did not get caught by our test suite because the relevant lines got normalized away.

Reply:
The stacktrace is not very helpful:
```
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:59
             at src/libstd/panicking.rs:197
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:211
   4: rustc::util::common::panic_hook
   5: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:478
   6: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:381
   7: rust_begin_unwind
             at src/libstd/panicking.rs:308
   8: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
   9: core::option::expect_failed
             at src/libcore/option.rs:1034
query stack during panic:
end of query stack
```

Reply:
I opened an issue for this: https://github.com/rust-lang/rust/issues/59774